packages/components/src/components/carousel-item/carousel-item.ts

```diff
@@ -0,0 +1,61 @@
+import { css, html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import componentStyles from '../../styles/component.styles';
+import SolidElement from '../../internal/solid-element.js';
+
+/**
+ * @summary A carousel item represent a slide within a [carousel](/components/carousel).
+ * @documentation https://solid.union-investment.com/[storybook-link]/carousel
+ *
+ * @since 1.12.0
+ * @status stable
+ *
+ * @slot - The carousel item's content.
+ *
+ *
+ */
+@customElement('sd-carousel-item')
+export default class SdCarouselItem extends SolidElement {
+  static isCarouselItem(node: Node) {
+    return node instanceof Element && node.getAttribute('aria-roledescription') === 'slide';
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.setAttribute('role', 'group');
+  }
+
+  render() {
+    return html` <slot></slot> `;
+  }
+
+  static styles = [
+    SolidElement.styles,
+    css`
+      ${componentStyles}
+      :host {
```

Comment:
Yeah, should be possible. We could either add them globally to the solid-element.ts (as we do with tailwind.css) or import them component per component.

Reply:
@mariohamann @yoezlem I might need a little more clarification here. Do we want to use tailwind syntax here instead of the vanilla CSS I already have? 

I am not exactly sure how to add these specific styles since they're selecting for the host and not the slot itself(?)

Reply:
Ah. Yeah so we don't have any styles that are relevant from our CSS styles here. So doing like you did is totally valid.